Client.xcodeproj/project.pbxproj

```diff
@@ -13386,6 +14619,7 @@
 					"$(BUILD_DIR)/Release$(EFFECTIVE_PLATFORM_NAME)",
 					"$(PROJECT_DIR)/ThirdParty/sqlcipher",
 				);
+				LOCALIZED_STRING_SWIFTUI_SUPPORT = NO;
```

Comment:
Looks like this invalidates all my comments about Text, might still be good to remember about localized vs verbatim